### DISCUSSION TITLE
Integrate Deno queue into Supabase

### DISCUSSION THREAD

**Discussion by ianschmitz** [supabase/supabase#18216] [Feature Requests]
Hi folks!

Deno recently launched [Deno Queues](https://deno.com/blog/queues). 

With Supabase's functions capability being built on Deno functions, it would be great to see queue functionality added via Deno queue.

Queues/async jobs are a common problem developers have to solve, and when using Supabase they have to use 3rd party solutions like SQS, Qstash, Inngest, etc. 

Thanks for considering.

**Comment by jcerjak** [comment]
Would be great to have some kind of queue functionality in Supabase. Deno Queues depend on Deno KV, not sure how that would work with Supabase, unless Supabase would support KV as well?

See also https://github.com/orgs/supabase/discussions/8508 for a related discussion on adding queue support to Supabase.

**Comment by bwship** [comment]
Yea, would definitely love this.  I think all they woul dneed to do is enable edge functions to automatically process the Deno Queues.  The only way to do this now would be with polling, but ideally it woudl work just like SQS with a Lambda attached to it in the AWS world.

**Comment by tazmaniax** [comment]
From the other discussion thread, have a look at https://sequinstream.com/ for PostgreSQL friendly queue alternative
